include/musica/tuvx/tuvx.hpp

```diff
@@ -66,15 +76,19 @@ namespace musica
     GridMap *GetGridMap(TUVX *tuvx, Error *error);
     ProfileMap *GetProfileMap(TUVX *tuvx, Error *error);
     RadiatorMap *GetRadiatorMap(TUVX *tuvx, Error *error);
+    void RunTuvx(TUVX *tuvx, const double solar_zenith_angle, const double earth_sun_distance,
+                 double * const photolysis_rate_constants, double * const heating_rates, Error * const error);
```

Comment:
I guess `const pointer to Error` is fine, but only this `Error` parameter is marked by `const` compared to the other functions that take `Error`
```C++
Error * const error
```

Reply:
Maybe we could make a follow-up task to go through all the code and use `const` consistently and where it makes sense?

Reply:
Yeah, I like the idea